### PR TITLE
Update metadata for kong-app

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/19663

Changes are

- Use SVG icon
- Use our own repo for the `home` attribute
- Assign team annotation
- Assign readme annotation (instead of oudated sources README URL)

### CODE REVIEW

File: helm/kong-app/Chart.yaml
Comment (webwurst) [inline helm/kong-app/Chart.yaml:14]:
```suggestion
version: 2.3.2
```

File: helm/kong-app/Chart.yaml
Comment (webwurst) [inline helm/kong-app/Chart.yaml:18]:
```suggestion
  application.giantswarm.io/readme: https://raw.githubusercontent.com/giantswarm/kong-app/v2.3.2/README.md
```

**Review by webwurst — CHANGES_REQUESTED** [review]
Confused why the version is off. Looks good apart from that.

File: helm/kong-app/Chart.yaml
Comment (ubergesundheit) [reply helm/kong-app/Chart.yaml:14]: Suggestion from @webwurst is correct. The `version` field is overwritten in CI by app-build-suite. The last one crafting a release forgot to update. For now this is manually. Unsolved ticket automating update is https://github.com/giantswarm/giantswarm/issues/16775

File: helm/kong-app/Chart.yaml
Comment (ubergesundheit) [reply helm/kong-app/Chart.yaml:18]: Should be removed entirely, app-build-suite overwrites this annotation anyway

**Comment by marians** [comment]
> @marians can you also update the https://github.com/giantswarm/kong-app/blob/master/CODEOWNERS file to match? Thanks

I think that should best be done via github.com/giantswarm/github

**Comment by ubergesundheit** [comment]
> I think that should best be done via github.com/giantswarm/github

It does not match the requirements imposed by giantswarm/github (needs word generated in the text)

**Comment by marians** [comment]
@ubergesundheit Yes. The existing CODEOWNERS file has to be removed, then via giantswarm/github a new one can be put in place.

**Comment by marians** [comment]
BTW, it's a bit tedious to come here and make changes to an app I don't "own" and then get asked for more changes, while the actual changes aren't even approved.

**Review by ubergesundheit — APPROVED** [review]

**Comment by marians** [comment]
@webwurst You are currently blocking this PR. Do you still want any changes?

**Review by webwurst — APPROVED** [review]

**Comment by webwurst** [comment]
I didn't intend to really block you. It's just that if I propose changes and approve at the same time it would look weird if the option of auto-merge would be active. So I clicked "Request changes". But anyone from the team approving would have been fine for me. Didn't know that GitHub sees this as a "veto". Which might be a valuable feature on the other hand..

**Pull request merged**
